docs/attributes-registry/url.md

```diff
@@ -26,9 +26,29 @@ linkTitle: URL
 
 **[2]:** The file extension is only set if it exists, as not every url has a file extension. When the file name has multiple extensions `example.tar.gz`, only the last one should be captured `gz`, not `tar.gz`.
 
-**[3]:** For network calls, URL usually has `scheme://host[:port][path][?query][#fragment]` format, where the fragment is not transmitted over HTTP, but if it is known, it SHOULD be included nevertheless.
-`url.full` MUST NOT contain credentials passed via URL in form of `https://username:password@www.example.com/`. In such case username and password SHOULD be redacted and attribute's value SHOULD be `https://REDACTED:REDACTED@www.example.com/`.
-`url.full` SHOULD capture the absolute URL when it is available (or can be reconstructed). Sensitive content provided in `url.full` SHOULD be scrubbed when instrumentations can identify it.
+**[3]:** For network calls, URL usually has `scheme://host[:port][path][?query][#fragment]` format, where the fragment
+is not transmitted over HTTP, but if it is known, it SHOULD be included nevertheless.
+
+`url.full` MUST NOT contain credentials passed via URL in form of `https://username:password@www.example.com/`.
+In such case username and password SHOULD be redacted and attribute's value SHOULD be `https://REDACTED:REDACTED@www.example.com/`.
+
+`url.full` SHOULD capture the absolute URL when it is available (or can be reconstructed).
+
+Sensitive content provided in `url.full` SHOULD be scrubbed when instrumentations can identify it.
+
+In particular, query string values for the following keys SHOULD be redacted by default and replaced by the
+value `REDACTED`:
+
+* `AWSAccessKeyId`
+* `Signature`
+* `sig`
+* `X-Goog-Signature`
```

Comment:
Should we give some added context as to why these were added?  What was the decision process and what criteria should be used in the future to evaluate if something new should be added?

Reply:
+1 on put some clarification around the criteria (doesn't have to be perfect in this PR though).

I have a general concern regarding the process, for example, if this list got updated everyday, are we going to play the catch-up game from all language implementations, and do we publish security advisories and deprecate previous versions?

Reply:
I've added links now from each of these query string keys to the public rest APIs that use them. This is the initial criteria I'm thinking of: _known_ query string keys that are used by _known_ public rest APIs.